assignments/prototype-refactor.js

```diff
@@ -7,3 +7,171 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+// yesterdays code
+
+/*
+  === GameObject ===
+  * createdAt
+  * dimensions (These represent the character's size in the video game)
+  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+*/
+
+/* YESTERDAY GAMEOBJECT */
```

Comment:
These comments make it really easy to see the translation between prototypes and classes!